Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -546,20 +555,69 @@ void SVertexer::buildT2V(const o2::globaltracking::RecoContainer& recoData) // a
   }
 
   LOG(info) << "Collected " << mTracksPool[POS].size() << " positive and " << mTracksPool[NEG].size() << " negative seeds";
+  if (!mTPCCounts.empty()) {
+    const auto total = std::accumulate(std::begin(mTPCCounts), std::end(mTPCCounts), 0, [](auto v, auto const& t) { return v + t.second; });
+    const float avg = static_cast<float>(total) / static_cast<float>(mTPCCounts.size());
+    const auto [min, max] = std::minmax_element(std::begin(mTPCCounts), std::end(mTPCCounts), [](const auto& p1, const auto& p2) { return p1.second < p2.second; });
+    LOGP(info, "TPConly: Total copies {} of {} tracks ({:.2f}) With a minimum of {} and maximum {} copies", total, mTPCCounts.size(), avg, (*min).second, (*max).second);
+    std::string str{"         `-> {{ "};
+    for (unsigned int i{(*min).second}; i <= (*max).second; ++i) {
+      auto c = std::count_if(std::begin(mTPCCounts), std::end(mTPCCounts), [&i](const auto& t) { return t.second == i; });
+      str += fmt::format("{}: {}, ", i, c);
+    }
+    str += "}}";
+    LOGP(info, str);
+  }
 }
 
 //__________________________________________________________________
 bool SVertexer::checkV0(const TrackCand& seedP, const TrackCand& seedN, int iP, int iN, int ithread)
 {
+  // Fast rough cuts on pairs before feeding to DCAFitter, tracks are not in the same Frame or at same X
+  bool usesTPCOnly = (seedP.hasTPC && seedP.nITSclu < 1) || (seedN.hasTPC && seedN.nITSclu < 1);
```

Comment:
My comment above is wrong, should have checked wrong code, I see that hasTPC is set to true and the nITSClu obviously should not be set at all since the default -1 is good by construction. 

The comment about not checking hasTPC if nITSclu<1 should be fine.